src/core/types.h

```diff
@@ -147,6 +148,9 @@ struct ParamNames {
     std::vector<llvm::StringRef> args;
     llvm::StringRef vararg, kwarg;
 
+    std::vector<AST_Name*> arg_names;
+    AST_Name* vararg_name, *kwarg_name;
```

Comment:
ah ok, makes sense.  I think the comment sounds good :)  splitting them into separate structs probably makes more sense, but for now this fits in with our "stuff things into the AST until we get around to doing a bytecode system" strategy :P